ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json

```diff
@@ -108,8 +108,8 @@
           {
             "Name": "What should I know about participating in this study?",
             "Answers": [
-              "You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva (spit) or cheek swab collection kit to use at home.",
-              "Most participants will receive a saliva (spit) collection kit.  Babies and toddlers under the age of 3 will receive a cheek swab kit.",
+              "You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva or cheek swab collection kit to use at home.",
+              "Most participants will receive a saliva collection kit.  Babies and toddlers under the age of 3 will receive a cheek swab kit.",
```

Comment:
1. first paragraph should be like 
`"You won’t need to go to a clinic to take part. You can sign up on this website. We will mail you a saliva collection kit to use at home."`

Reply:
2. The second paragraph ("Most participants ...") should be deleted.

see [the ticket](https://broadworkbench.atlassian.net/browse/PEPPER-324#:~:text=Delete%20the%20entirety%20of%20the%202nd%20bulleted%20answer%20for%20the%20same%20question%20to%20remove%20%22Most%20participants%20will%20receive%20a%20saliva%20(spit)%20collection%20kit.%20Babies%20and%20toddlers%20under%20the%20age%20of%203%20will%20receive%20a%20cheek%20swab%20kit%22.)

Reply:
Thanks for feedback :), I have already updated with your comments